.devcontainer/postCreate.sh

```
@@ -0,0 +1,233 @@
+            "temporary": false
+        }
+    ]
+}'
+
+echo retrive ID for user gaaldornick
+USER_DORNICK=$(curl "${KEYCLOAK}/admin/realms/${REALM}/ui-ext/brute-force-user?first=0&max=101&q=&search=gaaldornick" \
+  --header "authorization: Bearer ${KEYCLOAK_ACCESS_TOKEN}" \
+  --header 'content-type: application/json' | jq -r '.[0].id')
+
+echo Assign role _admin to seldon
+curl -X POST ${KEYCLOAK}/admin/realms/${REALM}/users/${USER_SELDON}/role-mappings/realm/${ROLE_ADMIN}  \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)